src/diffbot.js

```diff
@@ -411,7 +436,37 @@ class Diffbot {
           + `&name=${encodeURIComponent(options.name)}`
           + '&delete=1';
 
-        return fetch(diffbot_url, 'POST');
+        return request(diffbot_url, 'POST');
+      },
+      /**
+      * Crawlbot data filtered by search API
+       * @param {Object} options The options
+       * @param {string} options.name Name of the crawl whose data you wish to download.
+       * @param {format} [options.format] Request format=csv to download the extracted data in CSV format (default: json). Note that CSV files will only contain top-level fields.
+       * @param {string} [options.type] Request type=urls to retrieve the crawl URL Report (CSV).
+       * @param {number} [options.num] Pass an integer value (e.g. num=100) to request a subset of URLs, most recently crawled first.
+       * @returns The crawl job's results
+       */
+      search: function(options) {
+        // TODO: Do I police the optional fields or leave the user to get a 400 error?
+        if (!options.name) {
+          throw new Error('missing name');
+        } else if (!options.query){
+          throw new Error('missing query');
+        }
+
+        let diffbot_url = `https://api.diffbot.com/v3/search?token=${this.token}`
+          + `&col=${encodeURIComponent(options.name)}`
+          + `&query=${encodeURIComponent(options.query)}`;
+
+        if (options.num) {
+          diffbot_url += `&num=${options.num}`;
+        }
+        else{
+          diffbot_url += '&num=all';
+        }
+
+        return axios.get(diffbot_url);
```

Comment:
Is this where the issue you mentioned is? Because there is already a diffbot.search function at the bottom of the file. In this case it's using axios directly and returning the response and not resolving the data?

I would rather try and keep the behaviour standard and predictable across all of the functions, so if the heylocal app needs the response and not the data, would we be able to just generate the request from in the heylocal code?  So basically just take the code here and put it in the app directly, or include it as a helper function if it's used a lot?